debezium-storage/debezium-storage-azure-blob/src/main/java/io/debezium/storage/azure/blob/history/AzureBlobSchemaHistory.java

```diff
@@ -71,46 +57,32 @@ public class AzureBlobSchemaHistory extends AbstractSchemaHistory {
             .withDisplayName("container name")
             .withType(ConfigDef.Type.STRING)
             .withWidth(ConfigDef.Width.LONG)
-            .withImportance(ConfigDef.Importance.HIGH);
+            .withImportance(ConfigDef.Importance.HIGH)
+            .required();
 
     public static final Field BLOB_NAME = Field.create(CONFIGURATION_FIELD_PREFIX_STRING + BLOB_NAME_CONFIG)
             .withDisplayName("blob name")
             .withType(ConfigDef.Type.STRING)
             .withWidth(ConfigDef.Width.LONG)
-            .withImportance(ConfigDef.Importance.HIGH);
+            .withImportance(ConfigDef.Importance.HIGH)
+            .required();
 
     public static final Field.Set ALL_FIELDS = Field.setOf(ACCOUNT_CONNECTION_STRING, CONTAINER_NAME, BLOB_NAME);
 
-    private final AtomicBoolean running = new AtomicBoolean();
-    private final FunctionalReadWriteLock lock = FunctionalReadWriteLock.reentrant();
-    private final DocumentWriter documentWriter = DocumentWriter.defaultWriter();
-    private final DocumentReader reader = DocumentReader.defaultReader();
-
     private volatile BlobServiceClient blobServiceClient = null;
     private volatile BlobClient blobClient = null;
-    private String container = null;
-
-    private String blobName = null;
-
-    private volatile List<HistoryRecord> records = new ArrayList<>();
+    private String container;
+    private String blobName;
 
     @Override
     public void configure(Configuration config, HistoryRecordComparator comparator, SchemaHistoryListener listener, boolean useCatalogBeforeSchema) {
         super.configure(config, comparator, listener, useCatalogBeforeSchema);
         if (!config.validateAndRecord(ALL_FIELDS, LOGGER::error)) {
-            throw new SchemaHistoryException("Error configuring an instance of " + getClass().getSimpleName() + "; check the logs for details");
+            throw new ConnectException("Error configuring an instance of " + getClass().getSimpleName() + "; check the logs for details");
```

Comment:
Can we use `DebeziumException` here and in other places too?

Reply:
Done.